src/analysis/normed_space/star/multiplier.lean

```diff
@@ -0,0 +1,446 @@
+/-
+Copyright (c) 2022 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux, Jon Bannon
+-/
+
+import algebra.star.star_alg_hom
+import analysis.normed_space.star.basic
+import analysis.normed_space.operator_norm
+import analysis.special_functions.pow
+import analysis.normed_space.star.mul
+
+/-!
+# Multiplier Algebra of a C⋆-algebra
+
+Define the multiplier algebra of a C⋆-algebra as the algebra (over `𝕜`) of double centralizers,
+for which we provide the localized notation `𝓜(𝕜, A)`.  A double centralizer is a pair of
+continuous linear maps `L R : A →L[𝕜] A` satisfying the intertwining condition `R x * y = x * L y`.
+
+There is a natural embedding `A → 𝓜(𝕜, A)` which sends `a : A` to the continuous linear maps
+`L R : A →L[𝕜] A` given by left and right multiplication by `a`, and we provide this map as a
+coercion.
+
+The multiplier algebra corresponds to a non-commutative Stone–Čech compactification in the sense
+that when the algebra `A` is commutative, it can be identified with `C₀(X, ℂ)` for some locally
+compact Hausdorff space `X`, and in that case `𝓜(𝕜, A)` can be identified with `C(β X, ℂ)`.
+
+## Implementation notes
+
+We make the hypotheses on `𝕜` as weak as possible so that, in particular, this construction works
+for both `𝕜 = ℝ` and `𝕜 = ℂ`. The reader familiar with C⋆-algebra theory may recognize that one
+only needs `L` and `R` to be functions instead of continuous linear maps, at least when `A` is a
+C⋆-algebra. Our intention is simply to eventually provide a constructor for this situation.
```

Comment:
This second and third sentences have nothing to do with the first, right? Could you split off to its paragraph, for emphasis?